compositor_render/src/transformations/shader/node.rs

```diff
@@ -11,10 +11,16 @@ pub struct ShaderNode {
     params_bind_group: wgpu::BindGroup,
     _custom_params_buffer: wgpu::Buffer,
     shader: Arc<Shader>,
+    clear_color: Option<wgpu::Color>,
 }
 
 impl ShaderNode {
-    pub fn new(ctx: &WgpuCtx, shader: Arc<Shader>, params: Option<&ShaderParam>) -> Self {
+    pub fn new(
+        ctx: &WgpuCtx,
+        shader: Arc<Shader>,
+        params: Option<&ShaderParam>,
+        clear_color: Option<wgpu::Color>,
```

Comment:
I feel a bit that this does not belong here, but can't find a better place to pass this.

Reply:
I don't want to specify this on pipeline creation since the same pipeline might be used with different clear colors. I couldn't find a better place to put it either